05_syntax_lab/01.py

```diff
@@ -1,5 +1,11 @@
-"""
-Write a program that reads 10 numbers from
-the user and prints the largest one
-"""
-
+#"""
```

Comment:
You don't need both # and """. Usually we use """ at the beginning of the file for general file description, and # for line comments throughout the script